RadeonAsmDebugger.sln

```diff
@@ -89,13 +89,13 @@ Global
 		{A877B375-583E-4649-9981-9D4B3F025738}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{A877B375-583E-4649-9981-9D4B3F025738}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{A877B375-583E-4649-9981-9D4B3F025738}.Release|Any CPU.Build.0 = Release|Any CPU
-		{B7222544-B13F-48C8-85E3-6BFD3D44702A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{B7222544-B13F-48C8-85E3-6BFD3D44702A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{B7222544-B13F-48C8-85E3-6BFD3D44702A}.Release|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
Changes in this file don't make sense. We simply moved this 3 lines 4 lines further. Please revert